frontend/packages/noobaa-storage-plugin/src/components/resource-providers-card/resource-providers-card.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardHelp,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { ResourceProviderItems, providerType } from './resource-providers-card-body';
+import './resource-providers-card.scss';
+
+const RESOURCE_PROVIDERS_QUERY = {
+  PROVIDERS_TYPES: ' NooBaa_providers_types',
+  UNHEALTHY_PROVIDERS_TYPES: 'NooBaa_unhealthy_providers_types',
+};
+
+const getProviderType = (provider) => _.get(provider, 'metric.type', null);
+const getProviderCount = (provider) => Number(_.get(provider, 'value[1]', null));
+
+const createProvidersList = (data) => {
+  const providers = _.get(data, 'data.result', null);
+  const providersList: providerType = {};
+  if (_.isNil(providers)) return null;
+  providers.forEach((provider) => {
+    providersList[getProviderType(provider)] = getProviderCount(provider);
+  });
+  return providersList;
+};
+
+const ResourceProviders: React.FC<ResourceProvidersCardProps> = ({
```

Comment:
```suggestion
const ResourceProviders: React.FC<DashboardItemProps> = ({
```
and `ResourceProvidersCardProps` type can be removed